Types of Contributions

```diff
@@ -0,0 +1,264 @@
+TYPES OF CONTRIBUTIONS
+
+In this chapter, we will discuss the types of contribution that open source projects involved.
+
+Majorly, there are two types of contributions. Which are; 
+Non-coding type of contribution
+Coding type of contribution
+
+Non-Coding Type Of Contributions
+
+Non-coding contributions are valuable contributions made to open source projects that do not involve writing or modifying code. These contributions are essential for the overall health and success of open source communities and projects. They help improve documentation, user experience, project management, and community engagement.
+
+Here are some common types of non-coding contributions
+
+Documentation
+Graphic design
+Blog writing
+
```

Comment:
[proofreading] Consider presenting the list in bullet point form and hyperlinking each item. It'll make it easier for users to navigate! 
```suggestion

- [Documentation](#Documentation)
- [Graphic Design](#Graphic Design)
- [Blog Writing](#Blog Writing)

```